runtime/compiler/env/j9methodServer.cpp

```diff
@@ -265,6 +265,14 @@ TR_ResolvedJ9JITaaSServerMethod::getResolvedPossiblyPrivateVirtualMethod(TR::Com
       if (std::get<2>(recv) && unresolvedInCP)
          *unresolvedInCP = false;
 
+      bool createResolvedMethod = true;
+
+      if (comp->compileRelocatableCode() && ramMethod && comp->getOption(TR_UseSymbolValidationManager))
```

Comment:
I would like to understand why we need these extra pieces of code in resolvedMethods that are created for the non-AOT frontend. If they are needed, have can we make sure we fixed all the places that need such fixes.
Also, in some cases I see a test on `comp->compileRelocatableCode()` and `comp->getOption(TR_UseSymbolValidationManager)`, while in other cases there is just a test on `comp->getOption(TR_UseSymbolValidationManager)`

Reply:
The reason we need this code in base versions of resolved methods is because the AOT version calls non-AOT version. The purpose of these checks is to not create these types of resolved methods in AOT unless SVM is enabled.
Not sure why some of them don't have `comp->compileRelocatableCode()` test, I just copied what is done in non-JITaaS implementation. 
I think I fixed all such places.